src/components/BooksList.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from './Book';
+
+const BooksList = props => {
```

Comment:
**REQUIRED**
The BooksList component should be a container component since it has access to the state of the application and passes the state to the children components. 
Representaionational components are only concerned about how the page looks and has no access to the state. They can be defined as functional components.

Reply:
BooksList is now in the containers directory